testsuite/MDAnalysisTests/test_selections.py

```diff
@@ -97,6 +97,10 @@ def test_writeselection_ndx(self):
         self._write_selection(name=self.ref_name)
         self._assert_indices()
 
+    def test_write_ndx_bad_mode(self):
```

Comment:
Can you make this a generic method in `_SelectionWriter` instead of having almost the same method in each class?